infra/production/influxdb/main.tf

```diff
@@ -0,0 +1,59 @@
+terraform {
```

Comment:
TLDR: Create an EC2 instance for InfluxDB and execute a script to install InfluxDB and the Cloudflare tunnel inside the EC2 instance.